packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -11,6 +11,7 @@ export const PathConstants = {
   AWSConfig: 'config',
   DeploymentSecretsFileName: 'deployment-secrets.json',
   AmplifyAdminDirName: 'admin',
+  UsageDataFileName: 'amplify-configuration.json',
```

Comment:
In order to test this flow, you need to change this to amplify-dev-configuration.json. You will also need to delete this file from your ~/.amplify folder.